### ISSUE TITLE
32bit TF2 server failing to update

### ISSUE THREAD

**Issue by Gnomesenpai** [CM2Walki/steamcmd#64]
Hey, im using your SteamCMD and TF2 as a base for my TF2 servers, i have one server that just keeps failing with this, the other servers are using the exact same custom container I have but this one keeps failing. The error 0x6A6 is drawing a blank on the old google.

`app_update 232250
 Update state (0x1) running, progress: 0.00 (0 / 0)
 Update state (0x81) verifying update, progress: 78.43 (862398393 / 1099587423)
src/clientdll/depotreconstruct.cpp (734) : pInfo->nNumWritesFinished == 0
assert_20241218183216_19.dmp[1595]: Uploading dump (out-of-process)
/tmp/dumps/assert_20241218183216_19.dmp

Error! App '232250' state is 0x6A6 after update job.
`

**Comment by Gnomesenpai** [comment]
Interestingly cleared out the steamapps folder besides the appmanifest and suddently it started updating, guessing it had junk data in there.

**Comment by FERASMANASEER** [comment]
https://github.com/CM2Walki/steamcmd/issues/64

**Comment by CM2Walki** [comment]
Yes. It's a weird bug with steamcmd. Sometimes you have to delete the `steamapps` directory to force an update. I'm thinking about making this part of the startup process of the image. However, I'm unsure about the drawbacks.
